boards/ezurio/bl54l15_dvk/board.yml

```diff
@@ -0,0 +1,49 @@
+board:
+  name: bl54l15_dvk
+  full_name: BL54L15 DVK
+  vendor: ezurio
+  socs:
+    - name: nrf54l10
+      variants:
+        - name: ns
+          cpucluster: cpuapp
+    - name: nrf54l15
+      variants:
+        - name: xip
+          cpucluster: cpuflpr
+        - name: ns
+          cpucluster: cpuapp
+runners:
+  run_once:
+    '--recover':
+      - runners:
+          - nrfjprog
+          - nrfutil
+        run: first
+        groups:
+          - boards:
+              - bl54l15_dvk/nrf54l15/cpuapp
```

Comment:
`              - bl54l15_dvk/nrf54l15/cpuapp/ns` should also be here and in each part below